README.md

```diff
@@ -318,7 +318,7 @@ The following type definitions are used in the `ReactSDKClient` interface:
 
 - `UserAttributes : { [name: string]: any }`
 - `User : { id: string | null, attributes: userAttributes }`
-- `VariableValuesObject : { [key: string]: boolean | number | string | null }`
+- `VariableValuesObject : { [key: string]: unknown }`

```

Comment:
Not sure if we should keep it `unknown`. will the user need to cast it or provide a type now? Will it break for the users who are already using it directly as any?